Data/Sys/GameSettings/GK4.ini

```diff
@@ -9,8 +9,59 @@
 [ActionReplay]
 # Add action replay cheats here.
 
+[Gecko]
+$16:9 Widescreen
+F6000001 80008180
+5403D97E E3E10038
+D2000008 00000002
+38600001 CBE10030
+60000000 00000000
+E0000000 80008000
+F6000001 80008180
+ED6A4824 ED080172
+D2000000 00000005
+3821FFFC 93E10000
+3FE03FAB 93E30000
+C1630000 ED2B0272
+ED6A4824 83E10000
+38210004 00000000
+E0000000 80008000
+F6000001 80008180
+ED4B5024 ECA90172
+D2000000 00000006
+3821FFF4 93E10000
+D9210004 3FE03FAB
+93E30000 C1230000
+ED4902B2 ED4B5024
+83E10000 C9210004
+3821000C 00000000
+E0000000 80008000
+F6000001 80008180
+5460063F 418201FC
+D200002C 00000006
+3C803F40 90810008
+C0210008 C0030000
+EC010032 D0030000
+C003000C EC010032
+D003000C 38800001
+60000000 00000000
+E0000000 80008000
+F6000001 80008180
+5460063F 418203A4
+D2000040 00000006
+3C803F40 90810008
+C0210008 C0030000
+EC010032 D0030000
+C003000C EC010032
+D003000C 38800001
+60000000 00000000
+E0000000 80008000
```

Comment:
Are you sure this code works on all versions of the game? If not, it should be in a six-character INI file.

Reply:
Wiki says this is region-free. https://wiki.dolphin-emu.org/index.php?title=Baten_Kaitos_Origins